network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/take_action.html

```diff
@@ -28,12 +28,14 @@ <h2 class="take-action-title tw-mb-12">{{ page.take_action_title }}</h2>
         </div>
         <div class="tw-w-full tw-px-8 large:tw-w-1/3 medium:tw-mb-12 medium:tw-mt-12 large:tw-mt-0 tw-flex tw-items-stretch">
             <div class="tw-bg-black tw-p-12 tw-dark">
-                <div
-                    class="join-us"
+                <div class="newsletter-signup-module"
+                    data-module-type="default"
+                    data-form-style="home-body"
                     data-form-position="body"
+                    data-button-text="{% trans "Sign up" context "Submit button for newsletter signup form" %}"
                     data-cta-header="{% trans "Help shape the future of the web for the public good." %}"
-                    data-cta-description="<p class='mb-3'>{% trans "Join our <b>Mozilla News</b> email list to get action alerts & internet tips right to your inbox." %}</p>"
-                    data-newsletter="{{ cta.newsletter }}">
```

Comment:
`cta` is not available on the homepage so this `data-newsletter` attribute basically does nothing